docs/packages/markdown/loader.js

```diff
@@ -155,14 +155,14 @@ module.exports = async function demoLoader() {
       this.addDependency(moduleFilepath);
       demos[demoName] = {
         module: moduleID,
-        raw: await fs.readFile(moduleFilepath, { encoding: 'utf-8' }),
+        raw: await fs.readFile(moduleFilepath, { encoding: 'utf8' }),
```

Comment:
It's an alias, but the default is the one used here https://github.com/nodejs/node/blob/2364ed7d669bd569f8a70147c177a132ef5a7100/lib/buffer.js#L688.